util/followthemoney_util/excel.py

```diff
@@ -0,0 +1,29 @@
+import os
+
+import click
+
+from followthemoney_util.cli import cli
+from followthemoney_util.util import read_object
+from followthemoney.export.excel import get_workbook, get_sheet, write_entity
+
+
+@cli.command('export-excel', help="Export to Excel")
+@click.option('--filename', default='export.xlsx',
+              help="name of the excel file")
+@click.option('--outdir', type=click.Path(exists=True), default='.',
```

Comment:
I feel this might be redundant with `filename`, perhaps we can use validation on that (i.e. make it a `click.Path`) and remove `outdir`.

Reply:
Yeah, makes sense.